x/fswap/genesis.go

```diff
@@ -0,0 +1,24 @@
+package fswap
+
+import (
+	sdk "github.com/Finschia/finschia-sdk/types"
+	"github.com/Finschia/finschia-sdk/x/fswap/keeper"
+	"github.com/Finschia/finschia-sdk/x/fswap/types"
+)
+
+// InitGenesis initializes the capability module's state from a provided genesis
+// state.
+func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
```

Comment:
We should consider `swapped` state.

Reply:
Do I need add InitGenesis in this pr? I do this work in another pr.